instructions.md

```diff
@@ -125,6 +126,36 @@ as soon as you open the pull request. We can then often arrange to accept your p
 A reviewer will shortly comment on and/or accept your pull request, following this [process](reviewing.md).
 If you don't see a response within 3 business days, please contact conformance@cncf.io.
 
+## Example Script
+
+Combining the steps provided here, the process looks like this:
+
+```
+$ k8s_version=vX.Y
+$ prod_name=example
+
+$ go get -u -v github.com/heptio/sonobuoy
+
+$ sonobuoy run --wait
+$ outfile=$(sonobuoy retrieve)
+$ mkdir ./results; tar xzf $outfile -C ./results
+
+$ mkdir -p ./${k8s_version}/${prod_name}
+$ cp ./results/plugins/e2e/results/* ./${k8s_version}/${prod_name}/ 
+
+$ cat << EOF > ./${k8s_version}/${prod_name}/PRODUCT.yaml
+vendor: Yoyodyne
+name: Turbo Encabulator
+version: v1.7.4
```

Comment:
May want to update version to say v1.14.1 as a more recent ref.

Reply:
That is the example version of their product they are certifying, not the k8s version. The script puts the k8s version in a named variable to try and make that clear too.